### PR TITLE
Rearrange head tag for performance

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

Rearrange head tag for performance

## QA

1. download this banch
2. ./run it
3. compare the performance compared to the current site

### CODE REVIEW

File: templates/templates/base.html [hunk 1]
```diff
@@ -2,6 +2,18 @@
 <!doctype html>
 <html class="no-js" lang="zh" dir="ltr">
 <head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+  <title>{% block title %}{{ title }}{% endblock %} | Ubuntu</title>
+
+  <link rel="stylesheet" type="text/css" media="screen" href="{% versioned_static 'css/styles.css' %}" />
```
Comment (sowasred2012) [inline templates/templates/base.html]: From the suggestion Harry gave us (check the comment on row 4, col 1 [here](https://docs.google.com/spreadsheets/d/1E6_drpCb8E0zQWazxPjbFn_zrFyQmk6aho3hjHF3oOE/edit?ts=5d96f993#gid=86320696)), I think this needs to be below the inline scripts, on line 25.

Comment (pmahnke) [reply templates/templates/base.html]: barry suggested in another PR that gtm is different... are there other scripts I am missing?

Comment (barrymcgee) [reply templates/templates/base.html]: Yes - the inline-scripts-before-css suggestion is for when the inline script will affect the styling of the DOM. We don't want to block delivery of CSS for GA tracking.

File: templates/templates/base.html [hunk 2]
```diff
@@ -2,6 +2,18 @@
...
+
+
+  <link rel="preconnect" href="https://www.google-analytics.com">
+  <link rel="preconnect" href="https://www.gstatic.com">
+  <link rel="preconnect" href="https://assets.ubuntu.com">
```
Comment (barrymcgee) [inline templates/templates/base.html]: This is superfluous as we preload fonts assets from the assets server nano-seconds later.

File: templates/templates/base.html [hunk 3]
```diff
@@ -53,7 +67,7 @@
   <!-- footer content goes here -->
   {% include "templates/footer.html" %}
 
-<script src="/static/dist/global-nav.js"></script>
+<script src="/static/dist/global-nav.js" async defer></script>
```
Comment (barrymcgee) [inline templates/templates/base.html]:
```suggestion
<script src="/static/dist/global-nav.js" defer></script>
```

You should use `async` or `defer` - in this case, use `defer`

**Pull request merged**